### ISSUE TITLE
Specify the EC2 instance that is started

### ISSUE THREAD

**Issue by lawrence-vo** [machulav/ec2-github-runner#106]
Is it possible to specify the EC2 instance that is started? I don't want to spin up a new EC2 instance each time a workflow is ran.

**Comment by Thermoflux** [comment]
I am in need of this as well. I need to start an instance I have already created because some software is licensed to the MAC and some hardware IDs.  The documentation doesn't show if this is supported yet. Can anyone shed light on how to do this.

**Comment by devenes** [comment]
I think it's easier than creating a new instance but I'm not sure if it's a very good way to use it as a runner like in a CI/CD process.

**Comment by sampath6851** [comment]
Yes i think it is easy creating the ec2 instances instead of going for other services.
